sentinel-adapter/sentinel-spring-webmvc-6x-adapter/src/test/java/com/alibaba/csp/sentinel/adapter/spring/webmvc/config/InterceptorConfig.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 1999-2019 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.adapter.spring.webmvc.config;
+
+import com.alibaba.csp.sentinel.adapter.spring.webmvc.SentinelWebInterceptor;
+import com.alibaba.csp.sentinel.adapter.spring.webmvc.SentinelWebTotalInterceptor;
+import com.alibaba.csp.sentinel.adapter.spring.webmvc.callback.BlockExceptionHandler;
+import com.alibaba.csp.sentinel.adapter.spring.webmvc.callback.RequestOriginParser;
+import com.alibaba.csp.sentinel.slots.block.BlockException;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+/**
+ * Config sentinel interceptor
+ *
+ * @author kaizi2009
+ */
+@Configuration
+public class InterceptorConfig implements WebMvcConfigurer {
+
+    @Override
+    public void addInterceptors(InterceptorRegistry registry) {
+        //Add sentinel interceptor
+        addSpringMvcInterceptor(registry);
+
+        //If you want to sentinel the total flow, you can add total interceptor
+        addSpringMvcTotalInterceptor(registry);
+    }
+
+    private void addSpringMvcInterceptor(InterceptorRegistry registry) {
+        //Config
+        SentinelWebMvcConfig config = new SentinelWebMvcConfig();
+
+        config.setBlockExceptionHandler(new BlockExceptionHandler() {
+            @Override
+            public void handle(HttpServletRequest request, HttpServletResponse response, BlockException e) throws Exception {
+                String resourceName = e.getRule().getResource();
+                //Depending on your situation, you can choose to process or throw
+                if ("/hello".equals(resourceName)) {
```

Comment:
Is it possible to avoid hardcoding?

Reply:
This is unit test code and this module code was copied from the `sentinel-spring-webmvc-adapter` module for temporary compatibility.